index.html

```diff
@@ -52,7 +52,8 @@ <h2>I'm a Software Developer</h2>
             </div>
         </section>
 
-        <section id="works" class="work_section flex_column">
+        <section id="works" class="flex_column work_section">
+            <div class="overlay"></div>
             <div class="work_head">
                 <h2 class="title">My Recent Works</h2>
                 <img src="assets/Vector.png" alt="vector" class="work_head_img">
```

Comment:
- [ ] Good job so far @BenM4k and @Paul-tes. You did a good job with the popup functionality. However, As per the project requirements and as the previous reviewer mentioned, the projects section needs to be created dynamically. :+1: I have attached a screenshot below.
![image](https://user-images.githubusercontent.com/102699958/231513024-d2bd2106-4953-4ce7-9436-9ec29140495c.png)

- [ ] This means the projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript. :wink: 
- You can check an example of this student's project: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95